consoles/sshVirtsh.pm

```diff
@@ -325,111 +325,169 @@ sub add_interface {
     return;
 }
 
+sub to_byte {
+    my $sp              = shift;
```

Comment:
It would be nice to use signatures in new functions, e.g.:
```suggestion
sub to_byte ($sp) {
```

Reply:
Yeah, sure. But wasn't there a reason behind to avoid signatures in Perl?

Reply:
Compatibility - but at this point we decided to drop support for too old Perl versions.

Reply:
yes, years ago. But to my understanding signatures are fine as long as we avoid some more exotic features.

Reply:
Maybe I am doing something wrong, but is if enable signatures with `use feature 'signatures'` I get warnings the feature is experimental

Reply:
I would recommend enabling them via `use Mojo::Base -strict, -signatures;` (or just add `, -signatures` to the `use` args where `Mojo::Base` is already used).

Reply:
> I would recommend enabling them via `use Mojo::Base -strict, -signatures;` (or just add `, -signatures` to the `use` args where `Mojo::Base` is already used).

Thanks, this was the magic. Good to know how to enable signatures in Perl. :)